addons/activities/functions/fnc_reverse.sqf

```diff
@@ -30,11 +30,12 @@ if (effectiveCommander _vehicle == _driver) then {
         _ec moveInCargo _vehicle;
         _ec setVariable ["grad_civs_virtual_ec", true];
     } else {
-        _ec = (crew _vehicle) select 1;
+        _ec = ((crew _vehicle) select {_x != _driver})#0;
```

Comment:
no idea how that slipped through. 

`crew` returns *only the driver* in civ vehicles